### ISSUE TITLE
Doesn't work on iOs 9+

### ISSUE THREAD

**Issue by arechsteiner** [willmcpo/body-scroll-lock#27]
I couldn't get this to work on my project so I started testing the example provided.

It seems that the body scroll lock does not work on iOS 9 and above, including the current 11. It works on iOS 8.

**Comment by Neddz** [comment]
I can confirm this does not work on iOS 11+ I tried it on a friend's device.

This might be related to events registering passively by default, I saw that here:

https://stackoverflow.com/questions/49500339/cant-prevent-touchmove-from-scrolling-window-on-ios

I don't have an iOS device so I can't test if this works.

**Comment by simshanith** [comment]
Need to switch from assigning to `ontouchmove` to [`EventTarget.addEventListener`](https://caniuse.com/#feat=addeventlistener) with `{passive: false}`

Wish I could use this today

**Comment by Neddz** [comment]
Sim's solution didn't work for me so I am proposing a new one.

The problem is that the body is passive so it allows scrolling on it even if the `targetElement` is shown as it is not waiting for a `preventDefault()` before scrolling. 

What I did was to change the document `touchdown` listener so it does not allow scrolling while stopping the propagation from the `targetElement` so it is not affected by the listener I created.

I tested it on a real device and it works.
